### ISSUE TITLE
Does not work with @apollo/client v3.0.0+

### ISSUE THREAD

**Issue by dvakatsiienko** [blackxored/apollo-link-logger#54]
* `apollo-link-logger` version: **1.2.3**
* `node` version: **12.18.2**
* `yarn` version: **1.22.4**

Relevant code or config

What you did: installed **@apollo/client v3.0.1** and **apollo-link-logger v1.2.3**

What happened:

Error occurred:

```js
Error: Cannot find module 'apollo-link'
```

Problem description:

Apollo client recently got a major 3.0.0 update, they moved `apollo-link` into the core `@apollo/client` package under the `@apollo/client/link` path.

Suggested solution:

Make `apollo-link` as direct dependency of `apollo-link-logger`, or setup connection for `apollo-link-logger` with new structure of `@apollo/client/link`.

Basically it is needed to change imported **ApolloLink** entity from 'apollo-link' to `@apollo/client`.

**Before:**
```js
/* eslint-disable import/no-unresolved,import/extensions */
import { ApolloLink } from 'apollo-link';
import formatMessage from './formatMessage';
import logging from './logging';

const loggerLink = new ApolloLink((operation, forward) => {
```

**After:**
```js
/* eslint-disable import/no-unresolved,import/extensions */
import { ApolloLink } from '@apollo/client';
import formatMessage from './formatMessage';
import logging from './logging';

const loggerLink = new ApolloLink((operation, forward) => {
```
